x-pack/solutions/security/plugins/security_solution/docs/testing/test_plans/detection_response/prebuilt_rules/rule_customization.md

```diff
@@ -0,0 +1,281 @@
+# Prebuilt Rule Customization Workflows
+
+This is a test plan for rule customization workflows specifically related to prebuilt rules
+
+Status: `in progress`.
+
+## Useful information
+
+### Tickets
+
+- [Test plan issue](https://github.com/elastic/kibana/issues/202068)
+- [Prebuilt rule customization](https://github.com/elastic/kibana/issues/174168) epic
+
+### Terminology
+
+- **Base version**: The version of the rule we ship with the rule package, can be thought of as the "original" version of the rule.
+
+- **Customized prebuilt rule**: A prebuilt rule that has been changed by the user from the base version of the rule. Also referred to as "Modified" in the UI.
+
+- **Non-customized prebuilt rule**: A prebuilt rule that has no change from the base version of the rule.
+
+- **Custom rules**: A rule created by the user themselves
+
+### Assumptions
+
+- Rule package used will have all rule versions present (no missing base versions)
+
+## Scenarios
+
+### Editing prebuilt rules
+
+#### **Scenario: User can edit a non-customized prebuilt rule from the rule edit page**
+
+**Automation**: 1 cypress test.
+
+```Gherkin
+Given a space with at least one prebuilt rule installed
+And the rule is unmodified
+When a user edits the rule from the rule edit page to something different than the original version
+Then the rule is successfully updated
+And the ruleSource should be "external"
```

Comment:
Could we have separate scenarios for `ruleSource` instead of checking it in every test scenario?